lisa/analysis/base.py

```diff
@@ -52,6 +60,17 @@ def setup_plot(self, width=16, height=4, ncols=1, nrows=1, **kwargs):
         plt.tight_layout(h_pad=3.5)
         return figure, axes
 
+    def check_events(self, required_events):
+        """
+        :raises: RuntimeError if some events are not available
+        """
+        available_events = set(self._trace.events)
+        missing_events = set(required_events).difference(available_events)
+
+        if missing_events:
+            raise RuntimeError(
```

Comment:
May be worth adding a new `MissingTraceEventError` (as a subclass of `RuntimeError`), so we can extend the use of the decorator to tests, and we will get a clearer exception output when navigating logs.